.rat-excludes

```diff
@@ -383,6 +383,11 @@ stm32f1xx_hal_conf.h
 system_stm32f1xx.c
 startup_stm32f103xb.s
 
+# Black VET6 BSP - BSD License
+stm32f4xx_hal_conf.h
+system_stm32f4xx.c
+startup_STM32F40x.s
+
```

Comment:
This files are already in `.rat-excludes`. Note that CI does not apply it, just informs user of files without licenses (excluded or not!). Leaving here should not be a problem...

Reply:
Thanks, I did not think clear.
I took those files from some other bsp so it was kind of obvious they should be handle already.
Removed.